php/WP_CLI/Runner.php

```diff
@@ -334,6 +334,44 @@ private function _run_command() {
 	}
 
 	/**
+	 * Perform a command against a remote server over SSH
+	 */
+	private function run_ssh_command( $ssh ) {
+
+		$host = $ssh;
+		$path = '';
+		if ( false !== ( $key = stripos( $host, ':' ) ) ) {
+			$path = substr( $host, $key + 1 );
+			$host = substr( $host, 0, $key );
+		}
+
+		WP_CLI::debug( 'SSH host: ' . $host, 'bootstrap' );
+		WP_CLI::debug( 'SSH path: ' . $path, 'bootstrap' );
+
+		$is_tty = function_exists( 'posix_isatty' ) && posix_isatty( STDOUT );
+
+		$wp_binary = 'wp';
```

Comment:
No, I didn't. Also, perhaps the PHAR isn't even aliased as `wp` on the target environment. I think that there will ultimately need to be a way to provide an alternate path.